unix_integration/src/resolver.rs

```diff
@@ -393,7 +393,7 @@ where
         account_id: &Id,
         token: Option<UserToken>,
     ) -> Result<Option<UserToken>, ()> {
-        match self.client.unix_user_get(account_id).await {
+        match self.client.unix_user_get(account_id, token.clone()).await {
```

Comment:
Should be references to avoid the clone here. Can you do another pr to make `token: Option<&UserToken>`?